src/Contracts/Models/Status.cs

```diff
@@ -0,0 +1,11 @@
+﻿// FileCopyrightText: © 2021 - 2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+namespace Monai.Deploy.WorkflowManager.Contracts.Models
+{
+    public enum Status
```

Comment:
Do we want to share the same statuses from messaging lib?

Reply:
Ok, but there seems to be a lot of issues with the name "TaskStatus", so might be worth us changing it as there are a lot of ambiguous calls with System.Threading.Tasks.TaskStatus.

Also I'm guesssing we will still want to use this Status enum for the workflowInstance Status.

Reply:
Changed to use this and renamed it in that package.